projects/optic/src/commands/oas/captures/capture-storage.ts

```diff
@@ -6,7 +6,7 @@ import crypto from 'crypto';
 
 export async function captureStorage(
   filePath: string
-): Promise<[boolean, string]> {
+): Promise<[boolean, string, number]> {
```

Comment:
Can we change this into an object with keys? Easier to understand if it's keyed rather than a tuple for larger tuple sizes